crates/ide-assists/src/handlers/convert_comment_block.rs

```diff
@@ -78,21 +76,23 @@ fn line_to_block(acc: &mut Assists, comment: ast::Comment) -> Option<()> {
     // Establish the target of our edit based on the comments we found
     let target = TextRange::new(
         comments[0].syntax().text_range().start(),
-        comments.last().unwrap().syntax().text_range().end(),
+        comments.last()?.syntax().text_range().end(),
     );
 
+    // We pick a single indentation level for the whole block comment based on the
+    // comment where the assist was invoked. This will be prepended to the
+    // contents of each line comment when they're put into the block comment.
+    let indentation = IndentLevel::from_token(comment.syntax());
+
+    let cms =
+        comments.into_iter().map(|c| line_comment_text(indentation, c)).collect::<Vec<String>>();
```

Comment:
We can move this back into the closure below (that way it wont get executed unless the assist gets actually invoked)